core/chains/evm/client/rpc_client_test.go

```diff
@@ -45,6 +45,7 @@ func TestRPCClient_SubscribeNewHead(t *testing.T) {
 
 	chainId := big.NewInt(123456)
 	lggr := logger.Test(t)
+	var missingBothWSAndHTTPErr = errors.New("cannot dial rpc client when both ws and http info are missing")
```

Comment:
We should try minimizing the lifespan of a variable. This one is only used in a single test case. There is no need to define it globally for the testing function.